### DISCUSSION TITLE
preview window for react.

### DISCUSSION THREAD

**Discussion by Manvendra-div** [phcode-dev/phoenix#979] [Q&A] [answered]
how can we use the preview window for localhost:3000 which is react output port.

**Comment by abose** [comment]
External live previews are not supported at the moment. 
Created a new feature request to track this: https://github.com/phcode-dev/phoenix/issues/980 . 

Please upvote this discussion if anyone has this requirement.

**Comment by abose** [comment, marked as answer]
Custom live previews have been supported for over a year now. Forgot to update. Closing as done.

React specific advanced features are still WIP @ https://github.com/phcode-dev/phoenix/issues/2186
